src/Model/LroPagedResponseGo.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using AutoRest.Core.Model;
+
+namespace AutoRest.Go.Model
+{
+    /// <summary>
+    /// Represents a response from a long-running operation that's also a paged operation.
+    /// This is a bit of a hack as the "list all" method isn't part of the code model and doing
+    /// it this way was the path of least resistance (we should fix this eventually).
+    /// </summary>
+    class LroPagedResponseGo : Response
```

Comment:
>   class  [](start = 1, length = 9)

nit: missing modifier ? ( should be `public` or `internal` ? ). Also related to this, I see we tend to have all classes and methods marked as `public`. do we actually want everything public ?

Reply:
I will mark it as internal.  Generally speaking the pattern we should follow is that all types created by the AutoRest should be public types and anything else should be internal.  I will clean up the other types I added accordingly.